csrankings-a.csv

```diff
@@ -927,6 +927,7 @@ Alois C. Knoll,TU Munich,http://www6.in.tum.de/Main/Knoll,NOSCHOLARPAGE
 Alois Knoll,TU Munich,http://www6.in.tum.de/Main/Knoll,NOSCHOLARPAGE
 Alois Zoitl,JKU Linz,https://www.jku.at/linz-institute-of-technology/forschung/research-labs/cyber-physical-systems-lab/team/univ-prof-di-dr-alois-zoitl,OG52DQ8AAAAJ
 Alok N. Choudhary,Northwestern University,http://www.eecs.northwestern.edu/~choudhar,q0nuNrEAAAAJ
+Alon Cohen,Tel Aviv University,https://english.tau.ac.il/profile/alonco,shoYR_AAAAAJ

```

Comment:
EE, not CS. Please explain how they meet the inclusion criteria.

Reply:
The added faculty from EE (Alon Cohen) is allowed to supervise graduate students in computer science

Reply:
Confirming that I am more than 75% at TAU. 
The CS school allows ML related faculty to supervise research students, since these faculty publish predominantly in ML venues.